manager/controllers/advisory_detail.go

```diff
@@ -61,6 +61,11 @@ func AdvisoryDetailHandler(c *gin.Context) {
 		return
 	}
 
+	if filterInURL(c.Request.URL.String()) {
```

Comment:
I would rather use `isFilterInURLValid(c *gin.Context) bool`. When it returns "false" we can just call return:
~~~golang
if !isFilterInURLValid(c) {
   return
}
~~~
You can get url string from c and also log error and write status code accordingly directly in the method so we don't need to repeat it in all method callings.